pkg/cli/cli.go

```diff
@@ -356,6 +356,8 @@ func (c cli) buildRootCmd() *cobra.Command {
 
 	// kubebuilder init
 	rootCmd.AddCommand(c.newInitCmd())
+	// kubebuilder edit
+	rootCmd.AddCommand(c.newEditCmd())
```

Comment:
Commands are ordered alphabetically so that they are easy to find, please move this above `kubebuilder init`

Reply:
Done